cmd/schedulerclient/client.go

```diff
@@ -35,6 +36,12 @@ const (
 )
 
 func main() {
+	if err := mainNoExit(); err != nil {
+		log.Fatalf("error: %v", err)
+	}
+}
+
+func mainNoExit() error {
```

Comment:
Nit: can we give this a different name, like `runApp()`? This sounds weird.

Reply:
Sure thing